src/fswAlgorithms/orbitControl/lambertSolver/_UnitTest/test_lambertSolver.py

```diff
@@ -0,0 +1,321 @@
+        xTrueSol2 = Izzo.x[idxSol2]
+    else:
+        x = 0.
+        xSol2 = 0.
+        xTrue = 0.
+        xTrueSol2 = 0.
+
+    # make sure module output data is correct
+    paramsString = ' for solver={}, rev={}, time={}, eccentricity={}, angle={}, alignmentThreshold={}, ' \
+                   'accuracy={}'.format(
+                    str(p1_solver),
+                    str(p2_revs),
+                    str(p3_times),
+                    str(p4_eccs),
+                    str(p5_angles),
+                    str(p6_align),
+                    str(accuracy))
+
+    np.testing.assert_allclose(v1,
+                               v1True,
+                               rtol=0,
+                               atol=accuracy,
+                               err_msg=('Variable: v1,' + paramsString),
+                               verbose=True)
+
+    np.testing.assert_allclose(v2,
+                               v2True,
+                               rtol=0,
+                               atol=accuracy,
+                               err_msg=('Variable: v2,' + paramsString),
+                               verbose=True)
+
+    np.testing.assert_allclose(validFlag,
+                               validFlagTrue,
+                               rtol=0,
+                               atol=accuracy,
+                               err_msg=('Variable: validFlag,' + paramsString),
+                               verbose=True)
+
+    np.testing.assert_allclose(v1Sol2,
+                               v1TrueSol2,
+                               rtol=0,
+                               atol=accuracy,
+                               err_msg=('Variable: v1Sol2,' + paramsString),
+                               verbose=True)
+
+    np.testing.assert_allclose(v2Sol2,
+                               v2TrueSol2,
+                               rtol=0,
+                               atol=accuracy,
+                               err_msg=('Variable: v2Sol2,' + paramsString),
+                               verbose=True)
+
+    np.testing.assert_allclose(validFlagSol2,
+                               validFlagTrueSol2,
+                               rtol=0,
+                               atol=accuracy,
+                               err_msg=('Variable: validFlagSol2,' + paramsString),
+                               verbose=True)
+
+    np.testing.assert_allclose(x,
+                               xTrue,
+                               rtol=0,
+                               atol=accuracy,
+                               err_msg=('Variable: x,' + paramsString),
+                               verbose=True)
+
+    np.testing.assert_allclose(xSol2,
+                               xTrueSol2,
+                               rtol=0,
+                               atol=accuracy,
+                               err_msg=('Variable: xSol2,' + paramsString),
+                               verbose=True)
+
+
+if __name__ == "__main__":
```

Comment:
if I run `python test_lambertSolver.py` I get an error message.
``` 
Traceback (most recent call last):
  File "/Users/hp/Repos/basilisk/src/fswAlgorithms/orbitControl/lambertSolver/_UnitTest/test_lambertSolver.py", line 29, in <module>
    from fswAlgorithms.orbitControl.lambertSolver._UnitTest.Support.IzzoLambert import *
ModuleNotFoundError: No module named 'fswAlgorithms'
```

Reply:
It runs fine on my computer and on CI, so I don't know where the issue lies. I rewrote this line to import IzzoLambert in a different way.